### PR TITLE
[pylint] Fix unspecified-encoding

### PR DESCRIPTION
**Pull request by pponnuvel**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [ ] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

File: sos/utilities.py [hunk 1]
```diff
@@ -275,7 +275,8 @@ def _check_poller(proc):
         else:
             expanded_args.append(arg)
     if to_file:
-        _output = open(to_file, 'w')  # pylint: disable=consider-using-with
+        # pylint: disable=consider-using-with
```
Comment (TurboTurtle) [inline sos/utilities.py:278]: This diable shouldn't be needed since we're now specifying the encoding, right?

**Review by TurboTurtle — COMMENTED** [review]
Looks good up to my one note.

File: sos/utilities.py (hunk 1 above)
Comment (pponnuvel) [reply sos/utilities.py:278]: That's a different warning (`consider-using-with`).

Comment (TurboTurtle) [reply sos/utilities.py:278]: Oh, duh. Sorry, went cross-eyed there I guess.

**Review by TurboTurtle — APPROVED** [review]

**Review by arif-ali — APPROVED** [review]

**Pull request merged**